API updates (mainly Ledger) (#1514)
corda/corda-docs-portal: content/en/platform/corda/5.0/developing-applications/ledger/_index.md

---
title: "Ledger"
date: 2023-06-08
version: 'Corda 5.0 Beta 4'
menu:
  corda5:
    identifier: corda5-fundamentals-ledger
    parent: corda5-develop
    weight: 4750
section_menu: corda5
---

# Ledger

A distributed ledger is a database of facts that is replicated, shared, and synchronized across multiple participants on a network. We call an entity that can start or be involved in a transaction a *participant*. 

> Participants are members in the same [application network]({{< relref "../../application-networks/_index.md" >}}), represented by virtual nodes. Participants require a public/private key pair: they are identified by their public key, and use the private key to sign transactions. In the current implementation in Corda 5.0 Beta, all participants must be members, and they use the ledger key that has been generated when joining the network.

Reviewer: ```suggestion
Participants are members in the same [application network]({{< relref "../../application-networks/_index.md" >}}), represented by virtual nodes. Participants require a public/private key pair: they are identified by their public key, and use the private key to sign transactions. In the current implementation in Corda 5.0 Beta, all participants must be members, and they use the ledger key that was generated when joining the network.
```
